src/styles/_variables.scss

```diff
@@ -0,0 +1,43 @@
+$mainColor: #155bd5;
+$secondColor: white;
+$textColor:#6d6d6d;
+$border-color:#e0e0e0;
+$border-color-blue:#1c3ab6;
+$border-color-grey:#efefef;
```

Comment:
Usually for themes it uses such kind of map. For instance, MUI contains a pallette object with all colors that can be used. They are sorted and grouped by context. Please use tools that provide sass. 

$theme-colors: (
  "success": #28a745,
  "info": #17a2b8,
  "warning": #ffc107,
);

.alert {
  // Instead of $theme-color-#{warning}
  background-color: map.get($theme-colors, "warning");
}